spec/models/feedback_response_map_spec.rb

```
@@ -25,8 +25,8 @@
     describe '#get_title' do
       it 'returns the correct title constant' do
         feedback_response_map = FeedbackResponseMap.new 
-        expect(feedback_response_map.get_title).to eq(ResponseMapSubclassTitles::FEEDBACK_RESPONSE_MAP_TITLE)
+        expect(feedback_response_map.get_title).to eq(ExpertizaConstants::ResponseMapTitles::FEEDBACK_RESPONSE_MAP_TITLE)
       end
     end
 
-end
\ No newline at end of file
+end
```

RuboCop (1.86.1)

[convention] Extra empty line detected at block body end.

(Layout/EmptyLinesAroundBlockBody)